rosidl_generator_py/cmake/rosidl_generator_py_get_typesupports.cmake

```diff
@@ -15,7 +15,7 @@
 macro(accumulate_typesupports)
   set(_typesupport_impl "")
   get_rmw_typesupport(_typesupport_impl ${rmw_implementation} LANGUAGE "C")
-  list(APPEND _typesupport_impls ${_typesupport_impl})
+  list_append_unique(_typesupport_impls ${_typesupport_impl})
```

Comment:
the package.xml already exports ament_cmake https://github.com/ros2/rosidl/blob/master/rosidl_generator_py/package.xml#L11

Can I just add `ament_export_dependencies(ament_cmake)` to fulfill this requirement?